PytHW1/Task1.py

```diff
@@ -0,0 +1,10 @@
+print('Введите число')
+a = int(input())
```

Comment:
Зачем использовать функцию `print` для вывода сообщения о вводе, когда этот инструмент заложен внутрь функции `input`?
```suggestion
a = int(input('Введите число '))
```